config/jsdoc/api/conf.json

```diff
@@ -1,36 +1,34 @@
 {
-    "opts": {
-        "recurse": true,
-        "template": "config/jsdoc/api/template"
```

Comment:
does it mean that the `config/jsdoc/api/template` directory can be removed or will it be used in a second step?

Reply:
In the medium term we‘ll probably remove it. Leaving it here for now for testing/comparison reasons.